examples/angular_bazel_architect/scripts/puppeteer-chrome-versions.js

```diff
@@ -0,0 +1,16 @@
+// Mapping of puppeteer releases to their default Chrome version
+// derived from https://github.com/puppeteer/puppeteer/blob/master/docs/api.md.
+// The puppeteer package.json file contains the compatible Chrome revision such as
+// "chromium_revision": "722234" but this does not map easily to the Chrome version
+// so we use this mapping here instead.
+module.exports = {
```

Comment:
ouch, feels like it belongs somewhere other than the users app, not sure where though